src/Expander/TC.hs

```diff
@@ -1,7 +1,8 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE MultiWayIf #-}
 {-# LANGUAGE ViewPatterns #-}
-module Expander.TC (unify, freshMeta, inst, specialize, varType, makeTypeMetas, generalizeType, normType) where
+{-# OPTIONS -Wno-unused-top-binds #-}
```

Comment:
There was a warning saying that `setTVLevel` is dead code because it is not used inside the file but also not exported. Because of its symmetry with `setTVKind` (now `setTVLinkage`), it seemed like a plausibly-useful function, so I decided to leave it in and to disable the warning.